bazel/google_cloud_cpp_deps.bzl

```diff
@@ -87,7 +87,7 @@ def google_cloud_cpp_deps():
             # committed to the main branch.
             patch_tool = "patch",
             patch_args = ["-p1"],
-            patches = [],
+            patches = ["googleapis.patch"],
```

Comment:
Maybe that could be `@com_github_googleapis_google_cloud_cpp//external:googleapis.patch`, dunno, just guessing.

Reply:
I'm not sure either, but given that `googleapis.patch` will never be merged my guess is that wouldn't work.